vault/policy_store.go

```diff
@@ -150,6 +150,11 @@ path "sys/tools/hash/*" {
 path "sys/control-group/request" {
     capabilities = ["update"]
 }
+
+# Allow a token to make requests to the Authorization Endpoint for OIDC providers.
+path "identity/oidc/provider/+/authorize" {
```

Comment:
Does this work for an already existing node/cluster? There should already be a default policy on storage so I'm not sure if it would get picked up.

Reply:
You're right that this doesn't get picked up by an existing instance. Looking closer to see how this can be accomplished.

Reply:
I wonder if we'll have to instruct those with existing instances to update their default policy? I'm not seeing any patterns in past changes to the default policy around upgrading/merging with what's in storage.

@briankassouf  - Do you happen to know what the typical path forward is for existing instances in this case?

Reply:
Result from discussion is that we'll write documentation to note that existing deployments should add this to a policy. Will keep this in the default policy for new deployments. Resolving.